cishouseholds/derive.py

```diff
@@ -872,3 +872,38 @@ def assign_correct_age_at_date(df: DataFrame, column_name_to_assign, reference_d
         + F.round((F.col("month_more") + F.col("day_more")) / 3, 0).cast("int"),
     )
     return df.drop("month_more", "day_more")
+
+
+def assign_days_since_think_had_covid_group(
+    df: DataFrame,
+    think_had_covid_column: str = "think_had_covid",
+    reference_column: str = "days_since_think_had_covid",
+    column_name_to_assign: str = "days_since_think_had_covid_group",
+) -> DataFrame:
+    """
+    Function to be applied for days_since_think_had_covid_group and
+    contact_known_or_suspected_covid_days_since_group. The variable
+    days_since_think_had_covid will give a number that will be grouped
```

Comment:
This description needs to be generic if applying to derive both variables. Currently referring to deriving one of the two variables